tests/lib/tools/simpleget

```diff
@@ -41,17 +42,50 @@ def main() -> None:
     # create a temp in the same directory as output
     outdir = os.path.dirname(output)
 
+    headers = {}
+    if opts.header:
+        for hdr in opts.header:
+            sp = hdr.split(": ")
+            if len(sp) != 2:
```

Comment:
I would use str.partition https://docs.python.org/3/library/stdtypes.html#str.partition which is somewhat like strings.Cut in Go